### DISCUSSION TITLE
can pion/webrtc be used in native client side? or just for server use?

### DISCUSSION THREAD

**Discussion by shushenghong** [pion/webrtc#2536] [General]
if pion/webrtc can be a substude for google libwebrtc for native client use, can anyone help me how to build pion/webrtc for native client use? like ios objectc + pion/webrtc ; android java jni + pion/webrtc; windows c#/c++ + pion/webrtc?

**Comment by kehiy** [comment]
I suggest using an implementation in your own language, the pion implementation is in go, so using it with another language should be suffering (if it possible)
